coder_sniffer/Drupal/Test/Semantics/ConstantNameUnitTest.php

```diff
@@ -9,7 +13,7 @@ class Drupal_Sniffs_Semantics_ConstantNameUnitTest extends CoderSniffUnitTest
      *
      * @return array The list of test files.
      */
-    protected function getTestFiles()
+  protected function getTestFiles($testFileBase)
```

Comment:
This change introduces a whitespace error. Strange that this is not being caught be the coding standards check.

Reply:
I am confused - didn't this change remove the whitespace error?

Reply:
I did end up fixing a bunch of phpcs things - maybe the rules changed slightly too?

Reply:
oh, sorry, I see what you are saying - hmmm. Oh, it's because files in the `*/Test/*` aren't run through as per the `phpcs.xml.dist`

Reply:
So I just ran the files in `*/Test/*` through phpcs and there are 'a lot' of errors in them (most historical) so I am thinking maybe that should be tackled in separate issue. I could fix up any that are introduced from this patch potentially if you like?

Reply:
No let's keep any major coding standards fixes for a followup, I don't want to make this more complicated than necessary :)